pkg/scaffold/resource/resource.go

```diff
@@ -74,7 +74,13 @@ func (r *Resource) Validate() error {
 			"version must match ^v\\d+(alpha\\d+|beta\\d+)?$ (was %s)", r.Version)
 	}
 
-	validationErrors := isDNS1035Label(strings.ToLower(r.Kind))
+	validationErrors := []string{}
+
+	if string(r.Kind[0]) == strings.ToLower(string(r.Kind[0])) {
```

Comment:
Is this line saying clearly enough "if Kind's first letter is lowercase"?

Kinda wanted to extract in its own function, but didn't think the benefit was worth the indirection 🤔 

@camilamacedo86 @Adirio @DirectXMan12

Reply:
I would say it is pretty clear but a comment before it saying that we check that the first letter is lower case would not hurt anyone.

Reply:
Fixed, thanks! Resolving this one.